### ISSUE TITLE
`prefers-reduced-transparency` CSS Media Query

### ISSUE THREAD

**Issue by lukewarlow** [w3ctag/design-reviews#881]
こんにちは TAG-さん!

I'm requesting a TAG review of CSS `prefers-reduced-transparency`.

The `prefers-reduced-transparency` media feature is used to detect if the user has requested the system minimize the amount of transparent or translucent layer effects it uses.

  - Explainer¹ (minimally containing user needs and example code): https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-transparency
  - Specification URL: https://drafts.csswg.org/mediaqueries-5/#prefers-reduced-transparency
  - Tests: https://github.com/web-platform-tests/wpt/blob/master/css/mediaqueries/prefers-reduced-transparency.html
  - User research: https://github.com/w3c/csswg-drafts/issues/443
  - Security and Privacy self-review²: https://gist.github.com/lukewarlow/f515062a03def6f214f7a1fad5446eb2
  - GitHub repo (if you prefer feedback filed there): n/a
  - Primary contacts (and their relationship to the specification):
      - Dean Jackson, Apple
      - [Florian Rivoal](https://github.com/frivoal), Invited Expert
      - [Tab Atkins Jr](https://github.com/tabatkins), Google
      - Daniel Libby, Microsoft
  - Organization(s)/project(s) driving the specification: 
       - [cookiecrook](https://github.com/cookiecrook) by implementing [this PR](https://github.com/w3c/csswg-drafts/pull/1709)
  - Key pieces of existing multi-stakeholder review or discussion of this specification:
      - [CSSWG GitHub Issue](https://github.com/w3c/csswg-drafts/issues/443)
      - [Intent to Prototype Firefox](https://groups.google.com/a/mozilla.org/g/dev-platform/c/l410J8odZjA/m/JdWVMNtSAQAJ)
      - [Intent to Prototype Chromium](https://groups.google.com/a/chromium.org/g/blink-dev/c/pWonDP0g9Q4/m/iSVel0aBBgAJ)
  - External status/issue trackers for this specification (publicly visible, e.g. Chrome Status):
      -  [Chrome Status](https://chromestatus.com/feature/5191066147356672)

Further details:

  - [x] I have reviewed the TAG's [Web Platform Design Principles](https://www.w3.org/TR/design-principles/)
  - Relevant time constraints or deadlines: n/a
  - The group where the work on this specification is currently being done: CSSWG
  - The group where standardization of this work is intended to be done (if current group is a community group or other incubation venue): CSSWG
  - Major unresolved issues with or opposition to this specification: adds a fingerprinting vector
  - This work is being funded by:

We'd prefer the TAG provide feedback as (please delete all but the desired option):

  💬 leave review feedback as a **comment in this issue** and @-notify @lukewarlow

**Comment by cookiecrook** [comment]
> Organization(s)/project(s) driving the specification:
> - @cookiecrook by implementing https://github.com/w3c/csswg-drafts/pull/1709

> Major unresolved issues with or opposition to this specification: adds a fingerprinting vector

Additional context on why this isn't implemented in WebKit yet found in [WebKit Position #145](https://github.com/WebKit/standards-positions/issues/145):

> ### Note on Fingerprinting
> 
> [ Several proposed media features, including this one] are subject to changes based on the 
> outcome of https://github.com/WebKit/standards-positions/issues/145 and other related privacy issues. We debated 
> not proposing https://github.com/w3c/csswg-drafts/issues/8651 until after privacy issues with @media/matchMedia() 
> had been resolved, but decided to propose it here anyway to further the discussion.

**Comment by LeaVerou** [comment]
Hi all,

We reviewed this during a breakout today.

We're happy to see this moving forward, so we will close this.
As part of the review we also looked at the [fingerprinting concerns raised by WebKit](https://github.com/WebKit/standards-positions/issues/145). Our opinion is that the user benefit outweighs the privacy concern. That said, exploring ways to mitigate this more generally without negatively impacting user experience, especially of users with accessibility needs, would be welcome.

Thank you for flying TAG!
